dir.props

```diff
@@ -14,6 +14,11 @@
     <RoslynPackageName>Microsoft.Net.ToolsetCompilers</RoslynPackageName>
   </PropertyGroup>
 
+  <!-- MSBuild will set NuGetRuntimeIdentifier to be 'win' so we pre set it correctly here -->
+  <PropertyGroup>
+    <NuGetRuntimeIdentifier Condition="'$(NuGetRuntimeIdentifier)'==''">None</NuGetRuntimeIdentifier>
```

Comment:
actually if it doesn't impact any library that doesn't need a RID I'm fine with just forcing the projects that do need a RID to specify it. I much prefer that then setting this here. 

The reason I'm pushing on this is that I want to strip down dir.props pretty far and move most of it to build.common.props and not have anything that isn't necessary in it.